fusion/python/sample.py

```diff
@@ -43,21 +46,23 @@
 # OAuth credential placeholders that must be filled in by users.
 # You can find them on
 # https://www.yelp.com/developers/v3/manage_app
-CLIENT_ID = None
-CLIENT_SECRET = None
-
+config = configparser.RawConfigParser()
+config.read('API_Keys.cfg')
```

Comment:
could you include a default `API_Keys.cfg` with bogus values?

Reply:
Also please don't forget to update it to use an API_KEY rather than CLIENT_ID / CLIENT_SECRET, more details [here](https://www.yelp.com/developers/documentation/v3/authentication#where-is-my-client-secret-going)